notifications-sample/README.md

```diff
@@ -0,0 +1,42 @@
+# Notifications Sample
+
+This sample showcases a handful of basic configurations for notifications in VS Code:
+- Info Notification
+- Info Notification as Modal
+- Warning Notification
+- Warning Notification with Actions
+- Progress Notification
+
+## Demo
+
+![demo](demo.gif)
+
+## VS Code API
```

Comment:
Should we link to the UX guidelines for notifications in this section?

Reply:
Good call, added 👍